woocommerce/woocommerce-android
# Receipt 6 - Replace custom html receipts with Stripe's receipts

Parent issue #4039 

Merge instructions
1. Make sure https://github.com/woocommerce/woocommerce-android/pull/4107 is merged
2. Remove "do not merge" label
3. Merge this PR

This PR removes all logic related to custom html receipts and replaces them with receipts generated by Stripe. Also adds support for emailing receipts.

I recommend review this PR by commits as otherwise the changes might look too big.

Known issues
- when the user clicks on "Print receipt" button it might take a few seconds before the Print UI gets shown -> we plan to introduce a progress indicator, but that's not part of this PR

To test:
1. Open detail of an unpaid order in US dollars
2. Tap "Collect payment" button
3. Walk through the connect + payment flows
4. Tap on "Send receipt" button and notice an email client with pre-filled content is shown
5. Go back to the app and tap on "Print receipt" button
6. Notice printing UI with Stripe's receipt is shown


Update release notes:
